app/src/main/java/ch/epfl/sweng/studyup/utils/adapters/SpecialQuestListViewAdapter.java

```diff
@@ -0,0 +1,64 @@
+package ch.epfl.sweng.studyup.utils.adapters;
```

Comment:
Good call on making this package, but why not putting all adapters in there? (listItemAdapter, ViewPagerAdapter)